config/kraft/controller.properties

```diff
@@ -31,21 +31,16 @@ controller.quorum.voters=1@localhost:9093
 
 ############################# Socket Server Settings #############################
 
-# The address the socket server listens on. It will get the value returned from
-# java.net.InetAddress.getCanonicalHostName() if not configured.
+# The address the socket server listens on. If not configured, the host name will be equal to the value of
+# java.net.InetAddress.getCanonicalHostName(), with PLAINTEXT listener name, and port 9092.
```

Comment:
Is this another case that we need to fix? I think we need to have the controller listener here. Perhaps we should even say that _only_ the controller listeners are allowed here when `process.roles=controller`?

Reply:
Agree!